data/data/aws/vpc/master-elb.tf

```diff
@@ -41,8 +41,9 @@ resource "aws_lb_target_group" "api_internal" {
   ), var.tags)}"
 
   health_check {
-    healthy_threshold   = 3
-    unhealthy_threshold = 3
+    # 2+1 times interval must be smaller than 35 sec the kube-apiserver waits
+    healthy_threshold   = 2
```

Comment:
You can leave `healthy_threshold` alone, right?  This is just about graceful API server shutdown.